test/runtests.py

```diff
@@ -111,19 +112,15 @@ def setUp(self):
 
         self.proc = None
         self.publisher = None
-        self.management_api = PulseManagementAPI(
-            management_url='http://{}:{}/api'.format(
-                pulse_cfg['host'], pulse_cfg['management_port']),
-            user=pulse_cfg['user'],
-            password=pulse_cfg['password']
-        )
-        self.guardian = PulseGuardian(self.management_api,
-                                      warn_queue_size=TEST_WARN_SIZE,
+        config.rabbit_management_url = 'http://{}:{}/api/'.format(
+            pulse_cfg['host'], pulse_cfg['management_port'])
+        config.rabbit_user = pulse_cfg['user']
+        config.rabbit_password=pulse_cfg['password']
```

Comment:
These are global settings that shouldn't change between tests; could they be set earlier, like in main() or setup_host()? (Similarly there was probably no good reason that the PulseManagementAPI was being constructed separately for each test. Another opportunity to fix things. :))